### DISCUSSION TITLE
Send two different ProgramChange by pressing and releasing the button

### DISCUSSION THREAD

**Discussion by santta** [tttapa/Control-Surface#457] [Q&A]
Hi, I want to implement one button toggle between two program changes.
To send PC 1 on pressing button,  PC 2 to release. 

who can help?

**Comment by tttapa** [comment]
You should be able to write something yourself using the [Custom-MIDI-Output-Element](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html) example. Also see the [`Control_Surface.sendPC()`](https://tttapa.github.io/Control-Surface-doc/Doxygen/d6/d72/classMIDI__Sender.html#ac409060e87ff64e9a4c2525f1746b64d) function.
